src/components/PremisesSelector.tsx

```diff
@@ -15,7 +15,7 @@ const PremisesSelector: React.FC<Props> = ({ onChange }) => {
     <select onChange={handleSubmit}>
       <option value="P->Q,P,~Q">Basic Example</option>
       <option value="P->Q">Modus Ponens</option>
-      <option value="~PvQ">De Morgen's Law</option>
+      <option value="~PvQ">De Morgens Law</option>
```

Comment:
You do want it to say "De Morgen's", though.  Not "De Morgens".

As the eslint error says,
```
`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.eslint(react/no-unescaped-entities)
```

Reply:
use `&apos;` in place of the apostrophe.